sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -5368,6 +5368,76 @@
               "optionsLabelPosition": "right",
               "lockKey": true
             },
+            {
+              "label": "Will your partner receive B.C. income assistance for persons with disabilities under the Employment and Assistance for Persons with Disabilities Act during your study period?",
+              "optionsLabelPosition": "right",
+              "inline": false,
+              "tableView": false,
+              "values": [
+                {
+                  "label": "Yes",
+                  "value": "yes",
+                  "shortcut": ""
+                },
+                {
+                  "label": "No",
+                  "value": "no",
+                  "shortcut": ""
+                }
+              ],
+              "validate": {
+                "required": true,
+                "onlyAvailableItems": true
+              },
+              "validateWhenHidden": false,
+              "key": "partnerHasBceaIncomeAssistance",
```

Comment:
Please change the name to `partnerHasBCEAIncomeAssistance`.

Reply:
Updated.